contrib/tools/lmc.md/lgraph-search.md

```diff
@@ -114,20 +114,23 @@ fmod GRAPH-FOLDING-SEARCH is
     op _<=_ : NodeSet NodeSet -> [Bool] [ditto] .
     ---------------------------------------------
     eq NS             <  NS'        = NS =/= NS' and-then NS <= NS' .
-    eq NS             <= NS         = true .
-    eq .NodeSet       <= NeNS       = true .
+    eq ND             <= ND         = true .
+    eq .NodeSet       <= NS         = true .
     eq NeNS           <= .NodeSet   = false .
     eq (NeNS ; NeNS') <= NeNS''     = NeNS <= NeNS'' and-then NeNS' <= NeNS'' .
    ceq ND             <= (ND' ; NS) = true if fold(ND, ND') :: Fold .
 
```

Comment:
Yeah, the overloading of `_<=_` is an unfortunate consequence of the choice to make the subsort `NodeId < Node`. Ideally this subsort would be removed, but that will have to wait because I suspect it will break things.

I've fixed the redundancy issue, otherwise.